apps/etterlatte-saksbehandling-ui/client/src/components/behandling/soeknadsoversikt/soeknadoversikt/virkningstidspunkt/utils.test.ts

```diff
@@ -0,0 +1,24 @@
+import { describe, expect, it } from 'vitest'
+import { hentMinimumsVirkningstidspunkt } from '~components/behandling/soeknadsoversikt/soeknadoversikt/virkningstidspunkt/utils'
+
+describe('minimumstidspunkt for virkningstidspunkt er', () => {
+  it('maaned etter doedsdato hvis soknad er mottatt innenfor tre aar', function () {
```

Comment:
```suggestion
  it('maaned etter doedsdato hvis soknad er mottatt innenfor tre aar', () => {
```
Veldig hva man liker selv, men i sånne tester har jeg inntrykk av at det er ganske vanlig å bruke arrow-syntax :)